isis/src/lro/apps/lrowac2isis/lrowac2isis.cpp

```diff
@@ -0,0 +1,833 @@
+      int numBands   = 2;
+
+      uveven->setDimensions(numSamples, numLines, numBands);
+      uveven->setPixelType(Isis::Real);
+
+      QString filename = baseFileName.path() + "/" + baseFileName.baseName() + ".uv.even.cub";
+      uveven->create(filename);
+
+      uvodd->setDimensions(numSamples, numLines, numBands);
+      uvodd->setPixelType(Isis::Real);
+
+      filename = baseFileName.path() + "/" + baseFileName.baseName() + ".uv.odd.cub";
+      uvodd->create(filename);
+    }
+
+    if(viseven && visodd) {
+      // padding[size-1] is max padding for vis (padding[0] or padding[4] or padding[6])
+      int numSamples = p.Samples();
+      int numLines   = (int)(visOutputLineRatio * inputCubeLines + 0.5) + padding[padding.size()-1];
+      int numBands   = ((uveven) ? padding.size() - 2 : padding.size());
+
+      viseven->setDimensions(numSamples, numLines, numBands);
+      viseven->setPixelType(Isis::Real);
+
+      QString filename = baseFileName.path() + "/" + baseFileName.baseName() + ".vis.even.cub";
+      viseven->create(filename);
+
+      visodd->setDimensions(numSamples, numLines, numBands);
+      visodd->setPixelType(Isis::Real);
+
+      filename = baseFileName.path() + "/" + baseFileName.baseName() + ".vis.odd.cub";
+      visodd->create(filename);
+    }
+
+    Pvl isis3VisEvenLab, isis3VisOddLab, isis3UvEvenLab, isis3UvOddLab;
+
+    TranslateLabels(pdsLab, isis3VisEvenLab, isis3VisOddLab, isis3UvEvenLab, isis3UvOddLab, ui);
+
+    writeNullsToFile();
+    p.StartProcess(separateFramelets);
+    p.EndProcess();
+
+    // Add original labels
+    OriginalLabel origLabel(pdsLab);
+
+    int numFramelets = padding.size();
+    PvlKeyword numFrameletsKeyword("NumFramelets", toString(numFramelets));
+
+    if(uveven) {
+      for(int grp = 0; grp < isis3UvEvenLab.groups(); grp++) {
+        uveven->putGroup(isis3UvEvenLab.group(grp));
+      }
+
+      History history("IsisCube");
+      uveven->write(history);
+      uveven->write(origLabel);
+
+      uveven->close();
+      delete uveven;
+      uveven = NULL;
+    }
+
+    if(uvodd) {
+      for(int grp = 0; grp < isis3UvOddLab.groups(); grp++) {
+        uvodd->putGroup(isis3UvOddLab.group(grp));
+      }
+
+      History history("IsisCube");
+      uvodd->write(history);
+      uvodd->write(origLabel);
+
+      uvodd->close();
+      delete uvodd;
+      uvodd = NULL;
+    }
+
+    if(viseven) {
+      for(int grp = 0; grp < isis3VisEvenLab.groups(); grp++) {
+        viseven->putGroup(isis3VisEvenLab.group(grp));
+      }
+
+      History history("IsisCube");
+      viseven->write(history);
+      viseven->write(origLabel);
+
+      viseven->close();
+      delete viseven;
+      viseven = NULL;
+    }
+
+    if(visodd) {
+      for(int grp = 0; grp < isis3VisOddLab.groups(); grp++) {
+        visodd->putGroup(isis3VisOddLab.group(grp));
+      }
+
+      History history("IsisCube");
+      visodd->write(history);
```

Comment:
Same here. `history.AddEntry();` has been removed between 274 and 275 due to segfault.